api/app/weather_models/process_grib.py

```diff
@@ -116,9 +116,10 @@ def open_grib(filename: str) -> gdal.Dataset:
     return gdal.Open(filename, gdal.GA_ReadOnly)
 
 
-def get_dataset_geometry(dataset: gdal.Dataset) -> (List[int], List[int]):
+def get_dataset_geometry(dataset: gdal.Dataset) -> List[float]:
```

Comment:
has this return type changed because what gdal is returning has changed?

Reply:
The return type hasn't changed, I just spotted that it was define incorrectly, and fixed it.